tests/pom.xml

```diff
@@ -0,0 +1,44 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.jenkins.jenkinsfile-runner</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.0-beta-5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>tests</artifactId>
+
+  <profiles>
+    <profile>
+      <id>launch-test</id>
+      <activation>
+        <property>
+          <name>environment</name>
+          <value>test</value>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>1.6.0</version>
+            <executions>
+              <execution>
+                <id>test</id>
+                <phase>test</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>make</executable>
```

Comment:
NIT: make is not really needed for the automated run. You could invoke tests directly

Reply:
Yes, but since JFR and CWP use make as tool I though it could be a good idea to keep it for coherence. We can just change it and migrate the makefile to shell scripts in a follow up if all of us agree it's a good idea.